src/test/cli/rbd/help.t

```diff
@@ -176,6 +176,7 @@
                    [--io-size <io-size>] [--io-threads <io-threads>] 
                    [--io-total <io-total>] [--io-pattern <io-pattern>] 
                    [--rw-mix-read <rw-mix-read>] --io-type <io-type> 
+                   [--pattern-byte <pattern-byte>] 
```

Comment:
`make check` is still pending, but I suspect the order here and in the options section below is wrong: after the move to `add_bench_common_options()`, I'd expect `--pattern-byte` to come before `--io-type`.